src/Access/UsersConfigParser.cpp

```diff
@@ -88,7 +88,12 @@ namespace
             {
                 auto role_id = UsersConfigParser::generateID(AccessEntityType::ROLE, role_name);
                 if (!allowed_role_ids.contains(role_id))
```

Comment:
Name `allowed_role_ids` is not quite precise anymore, let's rename it to `role_ids_from_users_config`?